code/modules/hydroponics/hydro_tools.dm

```diff
@@ -266,57 +265,30 @@
 	w_class = W_CLASS_SMALL
 	throw_speed = 2
 	throw_range = 10
-	var/toxicity = 4
-	var/pest_kill_str = 0
-	var/weed_kill_str = 0
-
-/obj/item/weapon/plantspray/weeds // -- Skie
+	var/toxicity = 40
+	var/uses = 0
+
+/obj/item/weapon/plantspray/proc/use(amount = 1)
```

Comment:
not your fault but since this is a refactor anyway, these plantsprays are overdue for being turned into normal spraybottles with actual reagents

Reply:
I'll do another PR to cover them, since it'll affect a lot more files